tests/queries/0_stateless/02415_all_new_functions_must_have_version_information.reference

```diff
@@ -185,6 +185,8 @@ atan
 atan2
 atanh
 bar
+base32Decode
```

Comment:
This is wrong. These functions should have version information, not be added to the ignore list.

Reply:
Sorry, I shall fix this.

Reply:
#80654